guides/source/active_record_querying.md

```diff
@@ -2256,6 +2256,24 @@ irb> assoc.unscope(:includes).pluck(:id)
 
 [`pluck`]: https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck
 
+### `pick`
+
+[`pick`][] can be used to query single or multiple columns from the underlying table of a model. It accepts a list of column names as an argument and returns the first one of the specified column values ​​with corresponding data type.
```

Comment:
I think this was copied and modified from the documentation of `pluck`.
The [explanation in the API docs](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pick) is a lot clearer to me.
What do you think about using some of the API docs wording?
For example:
```suggestion
[`pick`][] can be used to pick the value(s) from the named column(s) in the current relation. This is primarily useful when you have a relation that’s already narrowed down to a single row. It accepts a list of column names as an argument and returns the first one of the specified column values ​​with corresponding data type. 
```

PS. I'd also accept a seperate PR to reword the first paragraph of `pluck` to something more similar to the API docs.

Reply:
sounds good!
Indeed, I would change this as I thought the wording in the API docs was clearer.
And, I will create a separate PR for changes to `pluck`

Reply:
Supported by https://github.com/rails/rails/pull/48404/commits/900bf10890494ed1e3c6cc4f2ae05a5cbcdf6c6a

Changed the description to use some of the wording from the API documentation. Also, Added nuances to the main useful cases.